.github/workflows/conformance.yml

```diff
@@ -29,6 +29,6 @@ jobs:
       - name: install sigstore-python
         run: python -m pip install .
 
-      - uses: trailofbits/sigstore-conformance@0748d63c53810e36cc3f4bbe4114301080f0d844 # v0.0.3
+      - uses: sigstore/sigstore-conformance@0748d63c53810e36cc3f4bbe4114301080f0d844 # v0.0.3
```

Comment:
cc @di: I think this will fix the conformance suite, but I can't actually verify that without merging because of how `pull_request_target` workflows are updated.